OktaLoggerAppCenter.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "OktaLoggerAppCenter"
-  s.version          = "1.3.7"
+  s.version          = "1.3.11"
```

Comment:
curious why do we jump to 11? they're independent versions from each other podspecs.

Reply:
you're right we don't have to update this